README.md

```diff
@@ -375,6 +375,12 @@ and then reference identifier `rest-lookup-logger` in the HTTP lookup DDL proper
 is provided.
 
 
+- Callback Errors:
+
+   It is also possible to declare if a request should be considered failed from the [HttpPostRequestCallback](src/main/java/com/getindata/connectors/http/HttpPostRequestCallback.java) by throwing a 
```

Comment:
what code would throw this exception - it would be helpful to give more detail  .

I find the words difficult to parse.  
 _It is also possible to declare if a request should be considered failed from the_ 

Can we say something like :

Throw HttpPostRequestCallbackException to indicate that a request should be considered as failed in .... code. The reason you might want to do this is ..... The side effects of doing this are .....

Reply:
Updated the description, lmk if this is easier to read